cmd/watcher.go

```diff
@@ -0,0 +1,124 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/k8tz/k8tz/pkg/watcher"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	client     *kubernetes.Clientset
+	kubeConfig *rest.Config
+	err        error
+
+	infoLogger  *log.Logger
+	errorLogger *log.Logger
+
+	secretWatcher = watcher.NewWatcher()
+)
+
+var watcherCmd = &cobra.Command{
+	Use:    "watcher",
+	Hidden: true,
+	Short:  "Starts Kubernetes Secret Watcher",
+	Long:   ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		// get kubeconfig, either from kubeconfig arg or incluster kubeconfig
```

Comment:
we need to make creation of `rest.Config` more consistent with the webhook: https://github.com/k8tz/k8tz/blob/master/pkg/admission/admission.go#L63-L76